add documentation for running knative on IBM s390x/IBM ppc64le platforms (#6043)
knative/docs: docs/install/yaml-install/serving/install-serving-with-yaml.md

The following tabs expand to show instructions for installing a networking layer.
Follow the procedure for the networking layer of your choice:

!!! note
    **Only Kourier network plugin supported for IBM Z and IBM Power platform.** 

>     Follow the below steps to install kourier. Post installation, [Use this alternative documentation](./install-serving-with-yaml-on-IBM-Z-and-IBM-P.md) to patch envoy image since we do not have envoy image support on IBM Z and IBM Power platforms, we will patch the envoy image based on RedHat Maistra.

Reviewer: ```suggestion
    Follow the below steps to install Kourier. Post installation, patch the envoy image based on RedHat Maistra as described in [this link](./install-serving-with-yaml-on-IBM-Z-and-IBM-P.md).
```
